script.module.enum34/lib/enum/setup.cfg

```diff
@@ -0,0 +1,5 @@
+[egg_info]
```

Comment:
remove

Reply:
This file provided by the original author Ethan Furman
Will removing it break the enum package?

Reply:
no this is only needed to packages for pypi

Reply:
OK